src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Comparator;
```

Comment:
The order of this import statement violates the coding standards. Where should it be instead? 🤔